src/components/app-registration/AppRegistration/AppRegistration.scss

```
@@ -166,9 +166,9 @@
   background-color: var(--white);
   color: black;
   font-weight: var(--text-weight-bold);
-  border: 1px solid #999999;
+  border: 0.1rem solid #999999;
```

- Borders don't need to scale.
- Can replace color with a var
```suggestion
  border: 1px solid var(--gray-05);
```